relay/state.go

```diff
@@ -4,6 +4,9 @@ import (
 	"fmt"
 	"os"
 	"path/filepath"
+
+	vaultType "github.com/vultisig/commondata/go/vultisig/vault/v1"
+	"github.com/vultisig/vultisigner/common"
 )
 
 type LocalStateAccessorImp struct {
```

Comment:
You should update LocalStateAccessorImp to accept a vault `.bak` file when create. decrypt it , and save it in memory
And then update `GetLocalState` function read from memory Vault structure
Currently `LocalStateAccessorImp` it is trying to read a json file , which will not exist , because we delete it after we backup the vault